### DISCUSSION TITLE
Primefaces Ecuador Theme - Error org/primefaces/component/menu/BaseMenuRenderer on Wildfly 31 and Jakarta

### DISCUSSION THREAD

**Discussion by rgpaladines** [primefaces/community#1348] [PrimeFaces Templates] [answered]
Hi, I have a problem with Primefaces Ecuador Theme 7.0.0 and Jakarta, when I install the theme, I have this error "Caused by: java.lang.NoClassDefFoundError: Failed to link org/primefaces/ecuador/component/EcuadorMenuRenderer".
Additionally, my pages with ajax support are not working, whithout Ecuador theme, all working normally
Help me please, thanks.

**Comment by melloware** [comment]
Are you using the latest version of Ecuador available in your PrimeStore account?  Make sure you are using the Jakarta version of the theme not the javax version?

**Comment by rgpaladines** [comment]
Yes, I'm sure, I'm using "ecuador-theme-7.0.0-jakarta.jar" and changed the classes "javax." by "jakarta."

**Reply by melloware** [reply]
What server are you you using? I feel like you might have a class path issue.

**Reply by rgpaladines** [reply]
Hi, I'm using standalone on Wildfly 31.0.1 Final, and load the theme with module library, all with Primefaces 13.0.6

**Reply by melloware** [reply]
OK all of that should be fine I use Widlfly 31 as well.  OK and can you verify in the WAR you deployed to Wildfly that `org/primefaces/ecuador/component/EcuadorMenuRenderer` is actually inside the JAR in WEB-INF/lib?

**Reply by rgpaladines** [reply, marked as answer]
Yes, you are right, that is the problem.
My solution with a Maven EAR project
- install the jakarta theme via maven.  For example:
mvn install:install-file -Dfile=ecuador-theme-7.0.0-jakarta.jar -DgroupId=org.primefaces.themes -DartifactId=ecuador-theme-7.0.0-jakarta -Dversion=7.0 -Dpackaging=jar -Dclasifier=jakarta

- add maven dependency
		<dependency>
		    <groupId>org.primefaces.themes</groupId>
		    <artifactId>ecuador-theme-7.0.0-jakarta</artifactId>
		    <version>7.0</version>
		</dependency>

It's fine for me.

Thanks a lot
